### DISCUSSION TITLE
"[@types/dom-mediacapture-record]  error with typescript greather than 4.3.5

### DISCUSSION THREAD

**Discussion by specchiasoft** [DefinitelyTyped/DefinitelyTyped#58160] [Issues with a @types package]
I have a compatibility problem on an Angular 13 project which requires typescript >= 4.4.3. 
If I install typescript in any version greather than 4.3.5 I get these errors, Please update compatibility of @types/dom-mediacapture-record many thanks.

![err_typescript](https://user-images.githubusercontent.com/25741411/149311075-cfb5eff3-f8dc-4d52-b7f0-e627014d00a3.jpg)

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment]
Hi, @specchiasoft. The minimum typescript version is 4.5.

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/dom-mediacapture-record/index.d.ts#L6
